docker-compose.yml

```diff
@@ -1,34 +1,59 @@
-postgresql:
-  restart: always
-  image: sameersbn/postgresql:9.4-18
-  environment:
+version: '2'
+
+networks:
```

Comment:
Yep, its not "required". 
Not sure if we should keep it or remove it. Guess we can remove it for now and add it later as we see fit?